src/test/regress/spec/isolation_setup.spec

```diff
@@ -0,0 +1,82 @@
+session "s1"
+step "setup"
+{
+	CREATE OR REPLACE FUNCTION pg_catalog.citus_isolation_test_session_is_blocked_skip_self_local_blocks(pBlockedPid integer, pInterestingPids integer[])
```

Comment:
I really don't think it is a good idea to exclude self-blocking on *all* isolation tests. If we introduce an unintended self-blocking operation, we should be able to spot. 

I noted this [here](https://github.com/citusdata/citus/pull/5910#discussion_r858416884) earlier

```
But, as I noted earlier, we cannot merge this change to citus_isolation_test_session_is_blocked for all, but only need to re-create this function when needed with these extra filters. Or, could even introduce citus_isolation_test_session_is_blocked_skip_self_local_blocks UDF, where we override citus_isolation_test_session_is_blocked with citus_isolation_test_session_is_blocked_skip_self_local_blocks when needed
```` 

Why do you think we should do it on all tests?

Reply:
> I really don't think it is a good idea to exclude self-blocking on all isolation tests.

I agree.

> Why do you think we should do it on all tests?

We actually don't. This setup only creates 3 functions. Self blocking is excluded only when a test file executes `replace_isolation_tester_func_skip_self_local_blocks`. That change is reverted at the end of those files when a corresponding `restore_isolation_tester_func_skip_self_local_blocks` call is executed.

Reply:
Oh, I see, thanks for the clarification. At a quick glance, I thought we changed on all files given the volume of the changes.

How hard would it be to only scope this function (e.g., setup) only for `CREATE/DROP/REINDEX INDEX CONCURRENTLY`   permutations that we suspect flakiness?